packages/react-components/react-progress/src/components/ProgressBar/useProgressBarStyles.ts

```diff
@@ -112,12 +112,21 @@ const useBarStyles = makeStyles({
 
   error: {
     backgroundColor: tokens.colorPaletteRedBackground3,
+    '@media screen and (forced-colors: active)': {
+      backgroundColor: 'Highlight',
+    },
   },
   warning: {
     backgroundColor: tokens.colorPaletteDarkOrangeBackground3,
+    '@media screen and (forced-colors: active)': {
+      backgroundColor: 'Highlight',
+    },
   },
   success: {
     backgroundColor: tokens.colorPaletteGreenBackground3,
+    '@media screen and (forced-colors: active)': {
+      backgroundColor: 'Highlight',
+    },
```

Comment:
I think the background color is the same for all appearances, right? if so, you could remove this and just add it in the base style instead.